config/MezaCoreExtensions.yml

```diff
@@ -348,7 +348,6 @@ list:
   - name: PipeEscape
     repo: https://github.com/wikimedia/mediawiki-extensions-PipeEscape.git
     version: "{{ mediawiki_default_branch }}"
-    legacy_load: true
```

Comment:
`legacy_load` is a Meza instruction that means the extension must be added to LocalSettings.php in the old legacy format using `require_once()` rather than the new format using `wfLoadExtension( 'ExtensionName' )` https://www.mediawiki.org/wiki/Manual:Extensions/Installation_and_upgrade